jetbrains-core/src/software/aws/toolkits/jetbrains/utils/NotificationUtils.kt

```diff
@@ -24,6 +28,26 @@ fun Exception.notifyError(title: String = "", project: Project? = null) =
 fun notifyInfo(title: String, content: String = "", project: Project? = null, listener: NotificationListener? = null) =
         notify(Notification(GROUP_DISPLAY_ID, title, content, NotificationType.INFORMATION, listener), project)
 
+fun notifyError(title: String, content: String = "", project: Project? = null, action: AnAction) =
+        notify(Notification(GROUP_DISPLAY_ID, title, content, NotificationType.ERROR).addAction(action), project)
+
+/**
+ * Check whether AWS credentials is configured. Notify if not configured and return true, otherwise return false
+ */
+fun checkAwsCredentialsOrNotify(
+    project: Project,
+    title: String = message("aws.notification.title"),
+    content: String = message("aws.notification.credentials_missing")
+): Boolean = if (ProjectAccountSettingsManager.getInstance(project).hasActiveCredentials()) false else {
```

Comment:
Reading the PR, I had a hard time understanding what this function does, and what the return values meant, until looking through this function and function comment. I think it should be renamed to better indicate this.

It might be more intuitive to have a function that notifies when necessary (and returning nothing), and then we already have the `hasActiveCredentials` function

Reply:
Used a more intuitive method name and logics :)